src/transformers/models/funnel/tokenization_funnel_fast.py

```diff
@@ -88,21 +91,55 @@
 PRETRAINED_INIT_CONFIGURATION = {f"funnel-transformer/{name}": {"do_lower_case": True} for name in _model_names}
 
 
-class FunnelTokenizerFast(BertTokenizerFast):
+class FunnelTokenizerFast(PreTrainedTokenizerFast):
     r"""
-    Construct a "fast" Funnel Transformer tokenizer (backed by HuggingFace's *tokenizers* library).
-
-    [`FunnelTokenizerFast`] is identical to [`BertTokenizerFast`] and runs end-to-end tokenization: punctuation
-    splitting and wordpiece.
-
-    Refer to superclass [`BertTokenizerFast`] for usage examples and documentation concerning parameters.
+    Construct a "fast" Funnel Transformer tokenizer (backed by HuggingFace's *tokenizers* library). Based on WordPiece.
+
+    This tokenizer inherits from [*PreTrainedTokenizerFast*] which contains most of the main methods. Users should
+    refer to this superclass for more information regarding those methods.
+
+    Args:
+        vocab_file (*str*):
+            File containing the vocabulary.
+        do_lower_case (*bool*, *optional*, defaults to *True*):
+            Whether or not to lowercase the input when tokenizing.
+        unk_token (*str*, *optional*, defaults to *"<unk>"*):
+            The unknown token. A token that is not in the vocabulary cannot be converted to an ID and is set to be this
+            token instead.
+        sep_token (*str*, *optional*, defaults to *"<sep>"*):
+            The separator token, which is used when building a sequence from multiple sequences, e.g. two sequences for
+            sequence classification or for a text and a question for question answering. It is also used as the last
+            token of a sequence built with special tokens.
+        pad_token (*str*, *optional*, defaults to *"<pad>"*):
+            The token used for padding, for example when batching sequences of different lengths.
+        cls_token (*str*, *optional*, defaults to *"<cls>"*):
+            The classifier token which is used when doing sequence classification (classification of the whole sequence
+            instead of per-token classification). It is the first token of the sequence when built with special tokens.
+        mask_token (*str*, *optional*, defaults to *"<mask>"*):
+            The token used for masking values. This is the token used when training this model with masked language
+            modeling. This is the token which the model will try to predict.
+        clean_text (*bool*, *optional*, defaults to *True*):
+            Whether or not to clean the text before tokenization by removing any control characters and replacing all
+            whitespaces by the classic one.
+        tokenize_chinese_chars (*bool*, *optional*, defaults to *True*):
```

Comment:
Same here.

Reply:
But running `make repo-consistency` fails when it is in between back-ticks

Reply:
I think that was because you had a pair of double backticks maybe? In any case, please push the change so I can look at the failure in the CI and the cause.